### ISSUE TITLE
how change DEFAULT_DOMAIN to new value and save it

### ISSUE THREAD

**Issue by TechComet** [flutter/flutter-intellij#7675]
Hi,

how change DEFAULT_DOMAIN to new value and save it

I write all my domain app like com.mycompany.<myappname>
how change and save default DEFAULT_DOMAIN value

Unfortunately every time I need to change it from com.example to com.mycompany

this problem for all versions

**Comment by helin24** [comment]
I don't think we have a way to change the default and this is unlikely to be something we will design and add. If you are generating new projects frequently, you may want to instead use `flutter create --org <org>` on the command line and save that to use each time you are making a project.
